packages/nimble-components/src/table-column/enum-base/models/mapping-spinner-config.ts

```diff
@@ -1,10 +1,36 @@
+import { type ViewTemplate, html } from '@microsoft/fast-element';
 import { MappingConfig } from './mapping-config';
+import type { IconView } from './mapping-icon-config';
+import { spinnerTag } from '../../../spinner';
+
+export interface SpinnerView {
+    text?: string;
+}
+
+const createSpinnerTemplate = (textHidden: boolean): ViewTemplate<IconView> => {
```

Comment:
The template created here (which parallels the template created for icons in `mapping-spinner-config.ts`) is just for the spinner. It will be used as part of the group-header-view's template or as part of the cell-view's template, and those templates are what control whether or not the text is rendered next to the icon/spinner. This template just makes sure the accessibility (i.e. tooltip, aria-hidden, aria-label) are correct for the visualization part of the view.

Reply:
Ah, gotcha! I had completely missed that case was present in the matrix test. Looks good 👍